source/documentors/references/doc_style_guide.rst

```diff
@@ -0,0 +1,407 @@
+- **Images and Graphs**: Are visuals used effectively to support the text?
+
+Review and Edit
+===============
+- **Proofreading**: Is the document free of spelling and grammatical errors?
+- **Feedback**: Has it been reviewed by someone else for clarity and effectiveness?
+- **Revisions**: Have necessary changes been made based on feedback?
+
+Accessibility
+=============
+- **Readable Layout**: Is the layout easy to navigate?
+- **Accessibility Features**: Are there considerations for individuals with disabilities (e.g., alt text for images)?
+
+Apart from answering those questions, our document needs to follow these principles:
+
+- **Make it clear**: Is the purpose of the document well-defined? The best documentation doesn't try to sound fancy, complicated, or self-important. While what you are doing may be difficult, there is no reason to make it seem that way through documentation. Instead, focus on clarity and ease of reading. To this end:
+    * Use simple language.
+    * Keep sentences short.
+    * Be direct.
+
+- **Make it scannable**: Good documentation should allow users to scan through for just the information that they need. To this end:
+    * Organize the information based on what is most important to your users. Thinking about things from the user's standpoint is a vital step to making the documentation accessible.
+    * Use headings and subheadings strategically to separate the document into scannable sections.
+    * Use stylistic elements strategically to help you organize your document and highlight the most critical information.
+    * Use bolded text rather than capital letters to draw attention to key words and phrases.
+
+    Use directives such as ``:info:``, ``:warning:``, and ``:guilabel:`` to draw attention to important text and button labels. For more information about this, please check this Quick Reference Directives.
+
+- **Make it connect**: Good documentation connects to other resources. This connection helps users gain a firmer grasp on the material, adds credibility to your writing, and eliminates the need to "reinvent the wheel." To this end:
+    * Link to other Open edX documentation that is relevant to your topic and audience.
+    * Link to other external sites only if they provide information not contained in the first two sources. Use caution when connecting to external sites, as the content of those sites may unexpectedly change without warning.
+    * All links should be written as text links in context. Your final document should not include full URLs written out.
+
+Audience
+========
+
+Make sure your documentation is targeted to the right audience and organized under that audience’s section of the documentation. Each topic in the documentation is for one of the following audiences:
+
+
+- Educators
+- Site Operators
+- Developers
+
+Ensure that new topics are stored in the directory for the appropriate audience, under the source directory. For more information, see Documentation Audiences.
```

Comment:
Can you link this to the appropriate document or section?

Reply:
@jesgreco can you address this comment?

Reply:
Done!